support/src/main/java/org/jenkinsci/plugins/workflow/support/steps/build/BuildTriggerStep.java

```diff
@@ -85,7 +85,7 @@ public DescriptorImpl() {
         @Override public Step newInstance(StaplerRequest req, JSONObject formData) throws FormException {
             BuildTriggerStep step = (BuildTriggerStep) super.newInstance(req, formData);
             // Cf. ParametersDefinitionProperty._doBuild:
-            JSONArray params = formData.optJSONArray("parameter");
+            JSONArray params = JSONArray.fromObject(formData.get("parameter"));
```

Comment:
No, it may be null, which is handled on the next line and tested in `generateSnippetForBuildTriggerNone`.